src/Microsoft.TestPlatform.Common/Filtering/FastFilter.cs

```diff
@@ -181,8 +182,8 @@ internal void AddCondition(Condition condition)
                     conditionEncountered = true;
                     fastFilterOperation = condition.Operation;
                     filterPropertyName = condition.Name;
-
-                    filterHashSetBuilder.Add(condition.Value);
+                    var value = condition.Value?.ToLowerInvariant();
```

Comment:
nit: you could just have `filterHashSetBuilder.Add(condition.Value?.ToLowerInvariant())` instead of putting it in a `value` first.